.gitignore

```diff
@@ -1,12 +1,8 @@
-.idea
-.DS_Store
-build
-phpunit.xml
-Resources/doc/_build/*
-nbproject
-coverage
+/build
+/Resources/doc/_build
+/vendor
+/node_modules
+/.sass-cache
 composer.lock
-vendor
-node_modules
-.sass-cache
+phpunit.xml
 .php_cs.cache
```

Comment:
Hello !

I've used the same order than on others projects here... Do you want that I move `node_modules`, `.sass-cache` at the end of the file ?